tslint.json

```diff
@@ -69,7 +69,7 @@
     ],
     "no-empty": false,
     "no-inferrable-types": [
-      true,
+      false,
```

Comment:
why?

Reply:
If it's true, it complains about types for variables.
For example **message: string = ' '** will throw tslint error, and would recommend to remove the type string.